libd/src/d/parser/conditional.d

```diff
@@ -161,12 +161,18 @@ auto parseMixin(ItemType)(ref TokenRange trange) if(is(Mixin!ItemType)) {
 	trange.match(TokenType.Mixin);
```

Comment:
Considering this has only one call site, I'm not sure there is a point at keeping it in this form.